### ISSUE TITLE
Table column does not get centered when content has hyphen

### ISSUE THREAD

**Issue by Wikiwix** [rvben/rumdl#426]
### rumdl version

0.1.19

### Installation method

Other

### Bug description

This file

```markdown
# Test

| Stage         |
| :-----------: |
| Red           |
| Amber         |
| Orange        |
| Green         |
| Integral      |
| Post-integral |
```

does not get centered on format ("no issues found" message).

When I remove the hyphen in `Post-integral` the formatting behaves as expected though

### Steps to reproduce

```bash
echo '# Test

| Stage         |
| :-----------: |
| Red           |
| Amber         |
| Orange        |
| Green         |
| Integral      |
| Post-integral |
' | rumdl fmt -
```

### Expected behavior

Column gets centered

### Actual behavior

No changes done

### Configuration

```toml
[table-format]
# https://rumdl.dev/md060/
enabled = true
style = "aligned"
```

### Operating system

Windows

**Comment by rvben** [comment]
Thanks for raising this @Wikiwix. This should not happen anymore in 0.1.20. 🚀
